src/utilities/xml/resources/bcl/component_v3.1.xsd

```diff
@@ -4,7 +4,7 @@
 	<xs:annotation>
 		<xs:documentation>Component.xsd describes the components that are in the building component library or NREL analysis platform library.</xs:documentation>
 	</xs:annotation>
-	<xs:element name="schema_version" fixed="3.0">
+	<xs:element name="schema_version" fixed="3.1">
```

Comment:
New 3.1 schema is the former 3.0. Uses a strict `xs:dateTime` format since this is now what we correctly write on a new BCLXML::save / saveAs